src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -2949,7 +2949,7 @@ void MusicXMLParserDirection::direction(const QString& partId,
         } else if (_e.name() == "staff") {
             QString strStaff = _e.readElementText();
             staff_idx_t staff = _pass1.getMusicXmlPart(partId).staffNumberToIndex(strStaff.toInt());
-            if (staff >= 0) {
+            if (staff > 0) {
```

Comment:
I'm not so sure about this one. Seems better to replace `staff_idx_t` with `int` in this case.

Reply:
In master that entire condition is gone, so I guess its saafe to remove it here too

Reply:
Still that `track += staff * VOICES;` makes only sense when `staff > 0`, it just doesn't harm if it is 0